content/guides/references/migration-guide.md

```diff
@@ -30,15 +30,18 @@ When the first argument is an object:
   rather than `Cypress.Blob`.
 - `encoding`: Remove this property. It is no longer needed due to improved
   binary file handling in Cypress 9.0.
-- `mimeType`: Remove this property. It is no longer needed due to improved
-  binary file handling in Cypress 9.0.
+- `mimeType`: This property is unsupported in Cypress 9.3.0. If you need
```

Comment:
Are we prioritizing adding support for `mimeType` in the near future? if so, maybe we could say something like "Support for mimeType will be added in a future update."

Reply:
Added "Support will be added in a future update."